### ISSUE TITLE
Add support of Universal Links on iOS

### ISSUE THREAD

**Issue by eepetrash** [blynkkk/blynk-server#1202]
Create file named `apple-app-site-association` with the following content:
1. For Blynk app on blynk-cloud.com:
```
{
    "applinks": {
        "apps": [],
        "details": [
            {
                "appID": "YCN439KN4Y.cc.blynk.blynk",
                "paths": ["*resetPass*"]
            }
        ]
    }
}
```

2. Both Blynk and MyPlant support on qa.blynk.cc server:
```
{
    "applinks": {
        "apps": [],
        "details": [
            {
                "appID": "YCN439KN4Y.cc.blynk.blynk",
                "paths": ["*resetPass*"]
            },
            {
                "appID": "YCN439KN4Y.cc.blynk.demo",
                "paths": [ "*resetPass*" ]
            }
        ]
    }
}
```

Notes:
- don’t append .json to the apple-app-site-association filename.
- you can create a plain text file that uses the application/json MIME type and you don’t need to sign it
- upload file to the root of the web server or to the .well-known subdirectory.

More info:
https://developer.apple.com/library/archive/documentation/General/Conceptual/AppSearch/UniversalLinks.html

**Comment by eepetrash** [comment]
@doom369 updated `apple-app-site-association` content in ticket

**Comment by doom369** [comment]
Implemented in Blynk 2.0
